### PR TITLE
Landing Page redesign

### PR DESCRIPTION
**Pull request by kellan-simiotics**

### CODE REVIEW

File: frontend/pages/index.js [hunk 1]
```diff
@@ -94,33 +94,33 @@ const Homepage = () => {
 
   const { buttonReport } = useContext(AnalyticsContext);
 
-  useEffect(() => {
-    assets["background720"] = `${AWS_ASSETS_PATH}/background720.png`;
-    assets["background1920"] = `${AWS_ASSETS_PATH}/background1920.png`;
-    assets["background2880"] = `${AWS_ASSETS_PATH}/background2880.png`;
-    assets["background3840"] = `${AWS_ASSETS_PATH}/background3840.png`;
-  }, []);
+  // useEffect(() => {
```
Comment (zomglings) [inline frontend/pages/index.js:97]: Get rid off these unused comment blocks?

File: frontend/pages/index.js [hunk 2]
```diff
@@ -94,33 +94,33 @@ const Homepage = () => {
...
-    isLargerThan720px,
-    isLargerThan1920px,
-    isLargerThan2880px,
-    isLargerThan3840px,
-    backgroundLoaded720,
-    backgroundLoaded1920,
-    backgroundLoaded2880,
-    backgroundLoaded3840,
-  ]);
+  // useLayoutEffect(() => {
```
Comment (zomglings) [inline frontend/pages/index.js:104]: Get rid.

File: frontend/pages/index.js [hunk 3]
```diff
@@ -139,40 +139,41 @@ const Homepage = () => {
...
   }, [isInit, router]);
 
-  useLayoutEffect(() => {
-    const imageLoader720 = new Image();
-    imageLoader720.src = `${AWS_ASSETS_PATH}/background720.png`;
-    imageLoader720.onload = () => {
-      setBackgroundLoaded720(true);
-    };
-  }, []);
+  // useLayoutEffect(() => {
```
Comment (zomglings) [inline frontend/pages/index.js:142]: Ditto.

File: frontend/pages/index.js [hunk 4]
```diff
@@ -139,40 +139,41 @@ const Homepage = () => {
...
+  // }, []);
 
-  useLayoutEffect(() => {
-    const imageLoader1920 = new Image();
-    imageLoader1920.src = `${AWS_ASSETS_PATH}/background1920.png`;
-    imageLoader1920.onload = () => {
-      setBackgroundLoaded1920(true);
-    };
-  }, []);
+  // useLayoutEffect(() => {
```
Comment (zomglings) [inline frontend/pages/index.js:150]: Ditto.

File: frontend/pages/index.js [hunk 5]
```diff
@@ -139,40 +139,41 @@ const Homepage = () => {
...
+  // }, []);
 
-  useLayoutEffect(() => {
-    const imageLoader2880 = new Image();
-    imageLoader2880.src = `${AWS_ASSETS_PATH}/background2880.png`;
-    imageLoader2880.onload = () => {
-      setBackgroundLoaded2880(true);
-    };
-  }, []);
+  // useLayoutEffect(() => {
```
Comment (zomglings) [inline frontend/pages/index.js:158]: Ditto.

File: frontend/pages/index.js [hunk 6]
```diff
@@ -139,40 +139,41 @@ const Homepage = () => {
...
+  // }, []);
 
-  useLayoutEffect(() => {
-    const imageLoader3840 = new Image();
-    imageLoader3840.src = `${AWS_ASSETS_PATH}/background3840.png`;
-    imageLoader3840.onload = () => {
-      setBackgroundLoaded3840(true);
-    };
-  }, []);
+  // useLayoutEffect(() => {
```
Comment (zomglings) [inline frontend/pages/index.js:166]: Ditto.

File: frontend/pages/index.js [hunk 7]
```diff
@@ -139,40 +139,41 @@ const Homepage = () => {
...
+  //   const imageLoader3840 = new Image();
+  //   imageLoader3840.src = `${AWS_ASSETS_PATH}/background3840.png`;
+  //   imageLoader3840.onload = () => {
+  //     setBackgroundLoaded3840(true);
+  //   };
+  // }, []);
 
   const blueBackgroundColor = "#212698";
-  const lightOrangeColor = "#FF9473";
+  const lightOrangeColor = "#F56646";
```
Comment (zomglings) [inline frontend/pages/index.js:175]: Colors approved by @ishihssihshihishsi ?

**Review by zomglings — COMMENTED** [review]
Better to remove commented out code. If you ever need it again (unlikely), get it from git history.

**Review by Andrei-Dolgolev — APPROVED** [review]
Looks correct

**Pull request merged**